packages/fuel-gauge/src/bytes.test.ts

```diff
@@ -86,40 +85,61 @@ describe('Bytes Tests', () => {
   });
 
   it('should test bytes input [predicate-bytes]', async () => {
-    const wallet = await setup(1_000_000);
+    using launched = await launchTestNode({
+      contractsConfigs: [
+        {
+          deployer: BytesAbi__factory,
+          bytecode: BytesAbiHex,
+        },
+      ],
+      walletsConfig: {
+        amountPerCoin: 1_000_000,
+      },
```

Comment:
```suggestion
```

[Default](https://github.com/FuelLabs/fuels-ts/blob/7ae80b8a4af7a2b770bb687edf1d9db671cc4a60/packages/account/src/test-utils/setup-test-provider-and-wallets.ts#L33) `amountPerCoin` is `10_000_000_000` so you don't need to define this.

Reply:
755f8e04b